### PR TITLE
Docs/read plugin markdown

### PR DESCRIPTION
**Pull request by markkelnar**
When moving documentation for wp-graphql plugin to markdown in that repo, sidebar navigation in .com is building the sidebar navigation from wp docs. This includes the documents listed, the sections, order and pagination. Moving that info to a yaml file in wp-graphql with the markdown files.  The code here reads that file and uses document lists and order from the yaml. If a label/title is not in the yaml for a document, it will look for that title from the markdown file, then the existing wp content.

Can preview this branch running at the site https://wpgraphqldocs.gatsbyjs.io/

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -17,6 +17,38 @@ exports.flatListToHierarchical = (
...
+      path: "#",
+      parent: null,
+      target: null,
+    }
+    newNav.push(newItem)
+    node.section.items.forEach((item) => {
+      // If yaml item doesn't have title, look for same document from other title sources
+      if (null === item.label) {
+        let found_item = titles.find((elem) => elem.uri === item.uri)
+        item.label = found_item ? found_item.title : null
```
Comment (markkelnar) [inline src/utils/index.js]: Would be an item in the wide bar without visual text to click on. That would be a weird state, but sign of something missing in the docs/markdown if none of the three sources could find the doc info.

**Comment by markkelnar** [comment]
These are the before and after. The Mutations page prev/next navigation coming from WP menus is wrong in production. See here

Screen Shot 2021-04-30 at 9 58 36 PM

With using the yaml file for navigation order, the prev/next matches the sidebar.

Screen Shot 2021-04-30 at 10 01 17 PM

File: gatsby-config.js [hunk 2]
```diff
@@ -116,25 +116,26 @@ const plugins = [
...
+      name: "documentation-v1",
     },
   },
   {
     resolve: "gatsby-transformer-remark",
     options: {
       plugins: [
         {
-          resolve: `gatsby-remark-images`,
+          resolve: `gatsby-remark-copy-linked-files`,
```
Comment (markkelnar) [inline gatsby-config.js:136]: I changed this to support animate gif coming up in the debugging markdown page.

**Comment by jasonbahl** [comment]
@markkelnar this is great! Love it. 🙌🏻 

Love the details you're catching with this. The animated gifs being copied is a great find/fix!

And already the pagination works better than the current production pagination!

**Pull request merged**